backend/src/handlers/auth.go

```diff
@@ -128,21 +126,6 @@ func (srv *Server) adminMiddleware(next func(http.ResponseWriter, *http.Request)
 	})
 }
 
-type OrySession struct {
-	Active    bool
-	TokenHash string
-	Expires   time.Time
-}
-
-func hashCookieValue(cookieValue string) string {
```

Comment:
All of this was removed in favor of pulling the kratos session cookie at the Auth endpoint. Our Identity/Auth services will always be hosted at the same top level domain, so we can take full advantage of all the CSRF + Cookie protections. Generally will be avoiding caching things in memory unless needed. NATS (cache + MQ) should fix that soon